packages/components/src/InputFile/InputFile.tsx

```diff
@@ -36,6 +36,11 @@ export interface FileUpload {
    */
   readonly progress: number;
 
+  /**
+   * Url file was uploaded to, used to send in mutation to JO
+   */
+  uploadUrl?: string;
```

Comment:
we mark most of our props as `readonly` to ensure we don't randomly overwrite it some place else and for overall consistency as well

```suggestion
  readonly uploadUrl?: string;
```